projects/core/src/lib/operators/dialogs/statistics-plot/statistics-plot.component.ts

```diff
@@ -19,43 +39,116 @@ import {StatisticsDict} from '../../../backend/operator.model';
     changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class StatisticsPlotComponent implements OnInit, AfterViewInit, OnDestroy {
-    readonly allowedLayerTypes = [ResultTypes.RASTER];
+    readonly allowedLayerTypes = ResultTypes.LAYER_TYPES;
+
+    readonly RASTER_TYPE = [ResultTypes.RASTER];
 
-    readonly subscriptions: Array<Subscription> = [];
+    attributes$ = new ReplaySubject<Array<string>>(1);
+
+    isVectorLayer$: Observable<boolean>;
+
+    isRasterLayer$: Observable<boolean>;
 
     form: UntypedFormGroup;
-    outputNameSuggestion: Observable<string>;
 
-    constructor(
-        private formBuilder: UntypedFormBuilder,
-        private projectService: ProjectService,
-        private notificationService: NotificationService,
-    ) {
+    private subscriptions: Array<Subscription> = [];
+
+    constructor(private formBuilder: UntypedFormBuilder, private projectService: ProjectService) {
+        const layerControl = this.formBuilder.control(undefined, Validators.required);
```

Comment:
It would be good, if you work on it, to change it to the `TypedFormBuilder`.

Reply:
090786e